deploy/handler/operator.yaml

```diff
@@ -118,6 +122,10 @@ spec:
           imagePullPolicy: {{ .HandlerPullPolicy }}
           command:
           - manager
+          resources:
+            requests:
+              cpu: "5m"
```

Comment:
This does not look right. I have a feeling that with 5m, the application would be too slow to be useful. Have you tried running with 5m set as the limit, to see if it is able to run that way?

Reply:
Its the request, it gives at least 5m and it can breath and extend, 
before that it was guaranteed 0m
We can't give it 5m limit even for test because it does breath more than that, but we dont set the highest

The actual numbers i measured (without setting limits that will throttle it in case of CPU limits from what i understand)

```
plain start |   3m / 18Mi
steady state |  4m / 22Mi
```

we can see that even on the start it used 3m, and reached 4m on the stead state
I can try to set limit as 5 if you desire, but its not reflect the true case, just very corner case when it will get exact 5m and wont breath

Reply:
I know it is just a request. But it is not more useful than 0m if it does not guarantee the application being useful, and that would defy the purpose of this effort.

Setting the request value as limit is a simple way to confirm that the application will be able to serve even under the tight resource situation.

Reply:
cpu limit 5m or 10m

It works but:
the secret takes more time to be created (but its created after 2 minutes since pod creation)
nmstate-webhook will be on `ContainerCreating` for around 2.5 - 5 minutes more than usual because it doesnt have the secret that it needs IIUC.
Eventually it will find the secret and will be `Running`

with 30m
secret created after 80 seconds
webhook turned to Fully (1/1) Running after 2m40s 

I will increase it then to 30m
(also the KMP cert-manager change which is already merged)

unless you think otherwise

thanks

Reply:
Yeah, let's do 30m on both. It is not that high, considering that both knmstate and KMP are heavy projects (compared to CNIs)

Reply:
KMP counterpart https://github.com/k8snetworkplumbingwg/kubemacpool/pull/320

Reply:
Note: without limit it takes around 50 seconds total for the nmstate-webhook to become Running